docs/book/component-gallery/model-deployers/kserve.md

```diff
@@ -218,6 +218,18 @@ pytorch_model_deployer = kserve_model_deployer_step(
 )
 ```
 
+Within the `KServeDeploymentConfig` you can configure:
+   * the model name : the name of the model in the KServe cluster and in ZenML.
+   * the number of replicas : the number of replicas to deploy the model with
+   * the predictor type : the type of predictor to use for the model. The
+    predictor type can be one of the following: `tensorflow`, `pytorch`, `sklearn`, `xgboost`, `custom`.
+   * the resources to be allocated to the model : the resources to be allocated
+    to the model. This can be configured by passing a dictionary with the
+    `requests` and `limits` keys. The values for these keys can be a dictionary
+    with the `cpu` and `memory` keys. The values for these keys can be a string
+    with the amount of CPU and memory to be allocated to the model.
+
+
```

Comment:
```suggestion
Within the `KServeDeploymentConfig` you can configure:
   * the model name: the name of the model in the KServe cluster and in ZenML.
   * the number of replicas: the number of replicas with which to deploy the model
   * the predictor type: the type of predictor to use for the model. The
    predictor type can be one of the following: `tensorflow`, `pytorch`, `sklearn`, `xgboost`, `custom`.
   * the resources to be allocated to the model: This can be configured by passing a dictionary with the
    `requests` and `limits` keys. The values for these keys can be a dictionary
    with the `cpu` and `memory` keys. The values for these keys can be a string
    with the amount of CPU and memory to be allocated to the model.

```